odxtools/paramlengthinfotype.py

```diff
@@ -20,9 +20,16 @@
 
 @dataclass
 class ParamLengthInfoType(DiagCodedType):
-
     length_key_ref: OdxLinkRef
 
+    @property
+    def dct_type(self) -> DctType:
+        return "PARAM-LENGTH-INFO-TYPE"
+
+    @property
+    def length_key(self) -> "LengthKeyParameter":
+        return self._length_key
```

Comment:
PR is already complex as is, those kind of non-functional code changes make the review even harder, please revert and move them to their own PR

Reply:
I agree, and I should have reverted this before opening the PR. That said, moving this into a separate PR now that it is already reviewed would increase your workload instead of reducing it, because github would show this as clutter in the "diff since your last review" view?

Reply:
you have the habit of using force-push, that means I have to do full review anyway.

Reply:
sorry, I was not aware that github does not take kindly to that. if you prefer, I can push the fixups as separate patches and only squash them before merge...

Reply:
please do that from now on